modules/mobile/spec/request/disability_rating_request_spec.rb

```diff
@@ -91,6 +103,19 @@
           end
         end
       end
+
```

Comment:
This case may be better suited for an adapter spec but the adapter is slim enough where there wouldn't be a whole lot of test cases in there and It's probably important to be testing out other components like the serializer in scenerios such as an empty array of individual ratings

Reply:
Yeah, I think it makes sense to lean into request specs until it becomes unsustainable. If the adapter isn't doing very much, request specs should be sufficient.